news/_posts/2021-05-31-prestashop-1-7-8-0-beta-release.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: post
+title:  "PrestaShop 1.7.8.0 beta is open for feedback!"
+subtitle: "You can download it and test it"
+date:   2020-05-31 17:00:00
+authors: [ PrestaShop ]
+icon: icon-leaf
+tags:
+ - version
+ - beta
+ - minor
+ - releases
+ - 1.7.8
+---
+
+The first Beta version for PrestaShop 1.7.8.0 is now ready for you to test!
+
+We are very happy to announce today the release of the beta version of 1.7.8.
+This version includes many features and bug fixes, we count on you to test every nook and cranny and report any bugs you encounter.
+
+In the new features: we have the redesign of the translation system that facilitates export and translation of your modules and themes, a new custom header for multistore fans and many other improvements.
```

Comment:
"In the new features" sounds a bit weird. Maybe "New features:" or "Regarding new features" would be better. :)